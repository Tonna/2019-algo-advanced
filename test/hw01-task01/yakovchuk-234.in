30
1 1 1 3 4 5 3 3 4 5 5 7 9 1 4 13 12 1 12 12 8 9 1 20 12 9 3 3 22
10
24 5
8 5
8 7
28 3
20 7
1 5
27 4
19 6
21 2
20 4
