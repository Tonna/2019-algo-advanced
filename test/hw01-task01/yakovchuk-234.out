30
9
13
5
19
7
16
26
8
12
